scripts/release_npm.sh

```diff
@@ -35,6 +35,12 @@ fi
 CMD_UPDATE_VERSION="lerna version ${PACKAGE_VERSION} --yes --exact --force-publish --no-git-tag-version --no-push"
 CMD_PUBLISH_PACKAGES="lerna publish ${PACKAGE_VERSION} ${TAG} --yes --exact --force-publish --no-git-tag-version --no-push --no-verify-access --no-verify-registry"
```

Comment:
There is no `--exact`, `--force-publish`, `--not-git-tag-version`, `--no-push` flags available for `publish` command.
The `--no-verify-registry` flag has been removed as well: https://github.com/lerna/lerna/commit/65fc603451c9547b10d6e0fe0a2e999e35da95c9#diff-afd35d92053195d1f03497360645ac8b
Why does `--no-verify-access` is present, I think it's something that we want.

If it's something that you don't want to tackle in this PR, let's log an issue to revisit this command.

Reply:
Removed the options that I was really sure didn't belong to publish. I don't want to remove the others I am unsure of before testing it well.
BTW, `lerna publish` supports all options supported by `lerna version` https://github.com/lerna/lerna/tree/master/commands/publish#options